modules/validations/crossing_ways.js

```diff
@@ -435,6 +441,10 @@ export function validationCrossingWays(context) {
 
                 if (connectionTags) {
                     fixes.push(makeConnectWaysFix(this.data.connectionTags));
+                    let lessLikelyConnectionTags = tagsForConnectionNodeIfAllowed(entities[0], entities[1], graph, true);
+                    if (lessLikelyConnectionTags && JSON.stringify(connectionTags) !== JSON.stringify(lessLikelyConnectionTags)) {
```

Comment:
When a footway crosses a major road, it would be possible to offer a `crossing=no` connection as a less likely option. However, `crossing=no` doesn’t have a preset yet (openstreetmap/id-tagging-schema#548) and looks just like `highway=crossing` on the map, which could be confusing.

Reply:
I agree that the tagging-schema would need to support this first in order to make sense. Let's keep that option in mind for a later stage.